src/styles.scss

```diff
@@ -1,6 +1,5 @@
 /* You can add global styles to this file, and also import other style files */
 /* Importing Bootstrap SCSS file. */
-@import '~bootstrap/scss/bootstrap';
 
 html, body { height: 100%; }
```

Comment:
Check this out: https://github.com/necolas/normalize.css/
This will do the work.
I personally prefer to keep that file inside the repo and import into main CSS